forms/gridfield/GridField.php

```diff
@@ -83,9 +87,9 @@ class GridField extends FormField {
 	/**
 	 * Creates a new GridField field
 	 *
-	 * @param string $name
-	 * @param string $title
-	 * @param SS_List $dataList
+	 * @param string          $name
```

Comment:
Can we just use simple spacing for phpdoc? Most of the codebase doesn't follow aligned indentation.